database/populateSQL.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-console */
+const path = require('path');
+const { Pool } = require('pg');
+
+const pool = new Pool({
+  host: 'localhost',
+  database: 'SDC',
+});
+
+const pricePath = path.join(`${__dirname}/data_gen/priceData.csv`);
+const videoPath = path.join(`${__dirname}/data_gen/videoData.csv`);
+const sidebarPath = path.join(`${__dirname}/data_gen/sidebarData.csv`);
+
+const copyCSV = (pathToCopy, type) => {
+  const insertText = `COPY ${type}
+  FROM '${pathToCopy}'
+  DELIMITER ','
+  CSV HEADER;`;
+  return pool.query(insertText);
+};
+
+const seedPSQL = async () => Promise.resolve(pool.connect())
+  .then(() => pool.query('SELECT NOW()'))
+  .then((result) => {
+    console.log('DB connected at ', result.rows[0].now);
+    const createTables = `DROP TABLE IF EXISTS price;
```

Comment:
According to [sequelize docs](https://sequelize.org/master/manual/model-basics.html#table-name-inference), the model name is made plural by default.  So the issue you're running into with connecting to the appropriate tables is that price is 'prices' thanks to sequelize.  you can turn this off by setting freezeTableName: true in your model definition

Reply:
thanks!!